README.md

```diff
@@ -80,7 +80,7 @@ parseInt('5', 10) === 5 // true
 
 ## Combining Comparisons
 
-We can combine these comparisons together using `&&` (pronounced "and") and `||` ("or"):
+We can string together these comparisons using `&&` (pronounced "and") and `||` ("or"):
```

Comment:
This is odd, I didn't add these changes "string together changes".. only the other ones below about adding 'false'